docker-compose.yml

```diff
@@ -279,7 +279,10 @@ services:
       - diet-error-handler
     environment:
       << : *govuk-app
+      GOVUK_CONTENT_SCHEMAS_PATH: /govuk-content-schemas
```

Comment:
Do you know if you actually need this? I think it might only be used in a test environment for this app (I was only aware that Publishing API used this for reals), same for the draft one. It probably harks back to when weren't sure whether these images would be used for test environment.

If it is needed you could add it to https://github.com/alphagov/publishing-e2e-tests/blob/63e49354e8112d3936312ce59afee2f0c9a99ec6/docker-compose.yml#L3 and then it would be automatically set for all apps.

Reply:
Aha, thanks for mentioning that! I had got mixed up between publishing-api and content-store here. I'll fix this shortly.

Reply:
Fixed in #436.